server/sonar-server/src/test/java/org/sonar/server/organization/ws/CreateActionTest.java

```diff
@@ -61,16 +65,19 @@
   @Rule
   public ExpectedException expectedException = ExpectedException.none();
 
+  private Settings settings = new MapSettings()
```

Comment:
I suggest to rely on default settings:
```private Settings settings = new MapSettings(new PropertyDefinitions(CorePropertyDefinitions.all()));```

then the property `ORGANIZATIONS_ANYONE_CAN_CREATE` is overridden in tests only when needed.